pkgs/development/tools/continuous-integration/gitlab-runner/default.nix

```diff
@@ -14,6 +14,9 @@ buildGoModule rec {
     "-X ${commonPackagePath}.REVISION=v${version}"
   ];
 
+  # For patchShebangs
+  buildInputs = [ bash ];
```

Comment:
I don't understand why this is needed. Is it needed at runtime or build time? If buildtime it should go to `nativeBuildInputs`.

Reply:
Good question, without it, it was leaving the `/usr/bin/env bash` as is, with it the `patchShebangs` works, so it's runtime/not a `nativeBuildInput`

Reply:
I understand, thanks for the explanation